plugins/newsdownloader.koplugin/feeds.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> a properly designed data structure should be able to be represented in a simple DSL like json.

JSON is a pain imo. When manually editing config files I like to:

```lua
return {
    -- list your feeds here:
    { "http://www.pcworld.com/index.rss", limit = 2 },
    { "http://rss.cnn.com/rss/edition_world.rss", limit = 3 },
    -- set limit to "0" means no download, "-1" no limit.
    -- LOOK MA I QUICKLY DISABLED THE NEXT LINE
    --{ "http://feeds.reuters.com/reuters/UKSportsNews?format=xml", limit = 0 },
}
```

But JSON doesn't allow that. XML (and Lua, yaml, and traditional INI files, etc.) does.

Anyway, I'm also against XML. [I can read and write it fluently](https://github.com/Frenzie/vierde-martijn/blob/a26330ce5c2e185df50db15af26351046b5eb051/mart4.xml), but it's rather verbose for the kind of thing we're talking about.